test/unit/specs/components/BorrowerProfile/PoweredByLenders.spec.js

```diff
@@ -0,0 +1,92 @@
+import { render } from '@testing-library/vue';
+import PoweredByLenders from '@/components/BorrowerProfile/PoweredByLenders';
+
+const participants = {
```

Comment:
Could this also use the new mock data?

Reply:
We already have contributing lender on the page, why using mock data?

Reply:
I was referring to the storybook mock data in this changeset, though I guess we don't typically share mocks between storybook and jest, so disregard.